lib/file-list.js

```diff
@@ -172,18 +173,18 @@ class FileList {
     const excluded = this._findExcluded(path)
     if (excluded) {
       log.debug(`Add file "${path}" ignored. Excluded by "${excluded}".`)
-      return this.files
+      return
```

Comment:
@devoto13 
I think not returning the file list is a breaking change. Should we not hold on to this change for release v7+ (https://github.com/karma-runner/karma/issues/3503)?

Reply:
@jginsburgn Yes, I think it should be better to land it in a major release.

Reply:
sure